src/components/app/list-new/Constants.ts

```diff
@@ -66,61 +66,32 @@ export const DefaultAppNote = `## Describe this application\n\nDescribe this app
 export const DefaultHelmChartNote = `## Describe this helm chart\n\nDescribe this helm chart in a few words. The description could include the purpose, features, benefits, and target audience of your application. A well-written description will help users know about this helm chart and how it works. Keep it concise and informative!\n`
 export const DefaultJobNote = `## Describe this job\n\nDescribe this job in a few words. The description could include the purpose, features, benefits, and target audience of your job. A well-written description will help users know about this job and how it works. Keep it concise and informative!\n`
 
+export const FLUX_CD_HELM_RELEASE_LABEL = 'Helm Release'
+
 export const StatusConstants = {
     NOT_DEPLOYED: {
         titleCase: 'Not Deployed',
         noSpaceLower: 'notdeployed',
         normalCase: 'Not deployed',
         lowerCase: 'not-deployed',
     },
-    APP_STATUS: {
-        noSpaceLower: 'appStatus',
-        normalText: 'App status',
-    },
-    PROJECT: {
-        pluralLower: 'projects',
-        lowerCase: 'project',
-    },
-    CLUSTER: {
-        pluralLower: 'clusters',
-        lowerCase: 'cluster',
-    },
-    NAMESPACE: {
-        pluralLower: 'namespaces',
-        lowerCase: 'namespace',
-    },
-    ENVIRONMENT: {
-        pluralLower: 'environments',
-        lowerCase: 'environment',
-    },
-    NOT_AVILABLE: {
-        normalCase: 'Not available',
-    },
-    TEMPLATE_TYPE: {
-        normalCase: 'Template type',
-        noSpaceLower: 'templateType',
-    },
 } as const
 
-export const InitialEmptyMasterFilters = {
-    projects: [],
-    environments: [],
-    clusters: [],
-    namespaces: [],
-    appStatus: [],
-    templateType: [],
-}
+export const TEMPLATE_TYPE_FILTER_OPTIONS: SelectPickerOptionType[] = [
+    { label: FLUX_CD_HELM_RELEASE_LABEL, value: FluxCDTemplateType.HELM_RELEASE },
+    { label: FluxCDTemplateType.KUSTOMIZATION, value: FluxCDTemplateType.KUSTOMIZATION },
+]
+
+export const APP_STATUS_FILTER_OPTIONS: SelectPickerOptionType[] = [
```

Comment:
Please verify on all references that we are doing structuredClone had seen few constants being passed directly